### PR TITLE
Drop all local tests except vyos

### PR DESCRIPTION
**Pull request by Qalthos**
This should be sufficient coverage for local

### CODE REVIEW

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -2072,16 +1977,6 @@
     vars:
       ansible_test_network_cli_ssh_type: libssh
 
-- job:
```
Comment (pabelanger) [inline zuul.d/jobs.yaml:2075]: Shouldn't we keep testing local on stable branches?  I think for vyos, we should be safe

File: zuul.d/jobs.yaml [hunk 2]
```diff
@@ -2098,16 +1993,6 @@
...
```
Comment (pabelanger) [inline zuul.d/jobs.yaml:2101]: same

File: zuul.d/project-templates.yaml [hunk 3]
```diff
@@ -597,13 +592,9 @@
     name: ansible-collections-vyos-vyos
     check:
       jobs: &ansible-collections-vyos-vyos-jobs
-        - ansible-test-network-integration-vyos-local-python27-stable210
```
Comment (pabelanger) [inline zuul.d/project-templates.yaml:600]: I think we should keep stable branches

Comment (Qalthos) [reply zuul.d/project-templates.yaml:600]: Did you mean this to be on 2.7, or the 3.6 runs below?

Comment (pabelanger) [reply zuul.d/project-templates.yaml:600]: yah, we still want to support python27 and local on stable branches. We've only dropped py27 on devel right now

Comment (Qalthos) [reply zuul.d/project-templates.yaml:600]: So... are we not dropping any local tests on vyos then?

Comment (pabelanger) [reply zuul.d/project-templates.yaml:600]: yah, I think we need to keep all on vyos, but can drop them every place else.

File: zuul.d/jobs.yaml [hunk 4]
```diff
@@ -1991,13 +1929,6 @@
     vars:
       ansible_test_python: 3.5
 
-- job:
```
[the remark this answers is not in the document]
Comment (Qalthos) [reply zuul.d/jobs.yaml:1994]:
```
% git grep "vyos-local-python27"
zuul.d/jobs.yaml:    name: ansible-test-network-integration-vyos-local-python27-stable210
zuul.d/jobs.yaml:    name: ansible-test-network-integration-vyos-local-python27-stable29
zuul.d/project-templates.yaml:        - ansible-test-network-integration-vyos-local-python27-stable210
zuul.d/project-templates.yaml:        - ansible-test-network-integration-vyos-local-python27-stable29
```
As you said, we stopped running 2.7 on devel, so why keep the job around? It's not even a parent for the stable jobs.

[the remark this answers is not in the document]
Comment (pabelanger) [reply zuul.d/jobs.yaml:1994]: oh, you are right. This is left over when I removed it yesterday

**Pull request merged**